test/Microsoft.ML.TimeSeries.Tests/TimeSeriesDirectApi.cs

```diff
@@ -276,5 +293,49 @@ public void ChangePointDetectionWithSeasonalityPredictionEngine()
             Assert.Equal(0.14823824685192111, prediction.Change[2], precision: 5); // P-Value score
             Assert.Equal(1.5292508189989167E-07, prediction.Change[3], precision: 5); // Martingale score
         }
+
+        [Fact]
```

Comment:
Is one test enough to ensure this new feature is functioning correctly?

Are there any negative/exceptional cases that need to be tested?

Reply:
This test case actually contains negative case, since it run anomaly detection once and assert both the anomalies and non-anomalies. I will add some cases for the parameter checking for the exceptional cases.